string name
uint8 type
--- 
string update_topic
